docker-plugin/src/main/java/io/micronaut/gradle/docker/MicronautDockerPlugin.java

```diff
@@ -63,19 +65,54 @@ public void apply(Project project) {
         dockerImages.all(image -> createDockerImage(project, image));
         TaskProvider<Jar> runnerJar = createMainRunnerJar(project, tasks);
         dockerImages.create("main", image -> {
+            createDependencyLayers(image, project.getConfigurations().getByName(RUNTIME_CLASSPATH_CONFIGURATION_NAME));
             image.addLayer(layer -> {
                 layer.getLayerKind().set(LayerKind.APP);
                 layer.getFiles().from(runnerJar);
             });
-            image.addLayer(layer -> {
-                layer.getLayerKind().set(LayerKind.LIBS);
-                layer.getFiles().from(project.getConfigurations().getByName(RUNTIME_CLASSPATH_CONFIGURATION_NAME));
-            });
-            image.addLayer(layer -> {
-                layer.getLayerKind().set(LayerKind.EXPANDED_RESOURCES);
-                layer.getFiles().from(project.getExtensions().getByType(SourceSetContainer.class)
-                        .getByName(SourceSet.MAIN_SOURCE_SET_NAME).getOutput().getResourcesDir());
-            });
+        });
+    }
+
+    public static void createDependencyLayers(MicronautDockerImage image, Configuration configuration) {
+        var projectLibs = configuration.getIncoming()
+            .artifactView(view -> {
+                view.lenient(true);
+                view.componentFilter(ProjectComponentIdentifier.class::isInstance);
+            }).getFiles();
+        var snapshotLibs = configuration.getIncoming()
+            .artifactView(view -> {
+//                view.lenient(true);
```

Comment:
is this commented out view.lenient(true) intentional?  if not set is it equivalent to `view.lenient(false)`?